sw/device/lib/arch/device.h

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_ARCH_DEVICE_H_
+#define OPENTITAN_SW_DEVICE_LIB_ARCH_DEVICE_H_
+
+#include <stdint.h>
+
+/**
+ * This header contains "device-specific" declarations, i.e., information that
+ * all devices are known to provide, but which is specific to the particular
+ * choice of platform, which can range from a software simulation, like
+ * Verilator or a DV testbench, to real harware, like an FPGA or ASIC.
+ *
+ * Definitions for these symbols can be found in other files in this directory,
+ * which should be linked in depending on which platform an executable is
+ * intended for.
+ */
+
+/**
+ * A |device_type_t| represents a particular device type for which
+ * device-specific symbols are available.
+ */
+typedef enum device_type {
+  /**
+   * Represents the "Verilator" device, i.e., a synthesis of the OpenTitan
+   * design by Verilator into C++.
+   */
+  kDeviceSimVerilator,
+  /**
+   * Represents the "Nexys Video FPGA" device, i.e., the particular FPGA board
+   * blessed for OpenTitan development, containing a Xilinx FPGA.
```

Comment:
blessed -> chosen? I personally don't mind quirky commenting style, but maybe someone does :)

Reply:
Dunno, I think "blessed" is more appropriate here, since it connotes that we specifically support this FPGA board. I'll change it if someone else disagrees.